src/rubberbandmodel.cpp

```diff
@@ -162,6 +163,9 @@ void RubberbandModel::setCurrentCoordinate( const QgsPoint& currentCoordinate )
   if ( mPointList.at( mCurrentCoordinateIndex ) == currentCoordinate )
     return;
 
+  if (mFreezed)
```

Comment:
Is there a pre-commit hook defined? I think I once copied the one from QGIS.

And sidenote, according to my English dict, it's "freeze, froze, frozen" ;)